Source/VMware.vSphereDSC/Tests/Unit/VMHostNtpSettings.Unit.Tests.ps1

```diff
@@ -687,9 +710,14 @@ Describe 'VMHostNtpSettings\Test' -Tag 'Test' {
             }
             $vmHostMock = {
                 return [VMware.Vim.VMHost] @{ ExtensionData `
-                     = [VMware.Vim.HostExtensionData] @{ Config = [VMware.Vim.HostConfig] @{ DateTimeInfo = [VMware.Vim.HostDateTimeConfig] @{ Id = 'HostDateTimeConfig'; NtpConfig `
-                     = [VMware.Vim.HostNtpConfig] @{ Server = @('0.bg.pool.ntp.org') } } }; ConfigManager = [VMware.Vim.HostConfigManager] @{ DateTimeSystem `
-                     = [VMware.Vim.HostDateTimeSystem] @{ Id = 'HostDateTimeSystem' } } } }
+                        = [VMware.Vim.HostExtensionData] @{ Config = [VMware.Vim.HostConfig] @{ DateTimeInfo = [VMware.Vim.HostDateTimeConfig] @{ Id = 'HostDateTimeConfig'; NtpConfig `
+                                    = [VMware.Vim.HostNtpConfig] @{ Server = @('0.bg.pool.ntp.org') }
```

Comment:
Here also the brackets could be on the same line and the same for the rest of the file.